README.md

```diff
@@ -32,88 +32,48 @@ git submodule update --init --recursive
+Open the root folder of this repository in Android Studio.
 
-```
-make aproj
-```
-
-#### linux
+### Project configuration
 
-run `make android-configuration` in the root folder of the project and open in Android Studio.
+#### Setup Mapbox Access token for dependency download
 
-If you are using Arch Linux, install [ncurses5-compat-libs](https://aur.archlinux.org/packages/ncurses5-compat-libs).
+Add a Mapbox access token with scope set to `DOWNLOADS:READ` in the root `build.gradle`.
 
-### Project configuration
+```groovy
+allprojects {
+    repositories {
+        maven {
+            url 'https://api.mapbox.com/downloads/v2/releases/maven'
+            authentication {
+                basic(BasicAuthentication)
+            }
+            credentials {
+                username = "mapbox"
+                password = "INSERT_MAPBOX_ACCESS_TOKEN_HERE"
+            }
+        }
+    }
+}
+```
 
 #### Setup Checkstyle
 
 Mapbox uses specific IDE settings related to code and check style.
 See [checkstyle guide](https://github.com/mapbox/mapbox-gl-native-android/wiki/Setting-up-Mapbox-checkstyle) for configuration details.
 
-#### Resolving duplicate file entries
-With buck build support, Android Studio can complain about duplicate source files. To remove this warning, open `MapboxGLAndroidSDK.iml` find the list of `excludeFolder` entries and add `<excludeFolder url="file://$MODULE_DIR$/../../../misc/" />` line.
-
-##### Setting Mapbox Access Token
+##### Setup Mapbox Access Token for Mapbox vector tiles
 
 _The test application (used for development purposes) uses Mapbox vector tiles, which require a Mapbox account and API access token. Obtain a free access token on the [Mapbox account page](https://www.mapbox.com/studio/account/tokens/)._
 
-With the first gradle invocation, gradle will take the value of the `MAPBOX_ACCESS_TOKEN` environment variable and save it to `MapboxGLAndroidSDKTestApp/src/main/res/values/developer-config.xml`. If the environment variable wasn't set, you can edit `developer-config.xml` manually and add your access token to the `mapbox_access_token` resource.  
-
-### Running project
-
-Run the configuration for the `MapboxGLAndroidSDKTestApp` module and select a device or emulator to deploy on. Based on the selected device, the c++ code will be compiled for the related processor architecture. You can see the project compiling in the `View > Tool Windows > Gradle Console`.
-
-More information about building and distributing this project in [DISTRIBUTE.md](DISTRIBUTE.md).
+With the first gradle invocation, gradle will take the value of the `MAPBOX_ACCESS_TOKEN` environment variable and save it to `MapboxGLAndroidSDKTestApp/src/main/res/values/developer-config.xml`. If the environment variable wasn't set, you can edit `developer-config.xml` manually and add your access token to the `mapbox_access_token` resource.
 
 ### Additional resources
 
-#### Using the SDK snapshot
```

Comment:
shall we ticket out the tail work to add snapshot back?

Reply:
yes: https://github.com/mapbox/mapbox-gl-native-android/issues/453 and https://github.com/mapbox/mapbox-gl-native-android/issues/454